src/vorta/views/extract_dialog.py

```diff
@@ -39,11 +37,11 @@ def parse_json_line(line):
                 d[name] = {}
             return size, modified, name, dirpath
 
-        for line in fs_data.split("\n"):
-            try:
-                files_with_attributes.append(parse_json_line(line))
-            except ValueError:
-                pass
+        # handle case of a single line of result, which will already be a dict
```

Comment:
What happens to the value error?

List comprehension will be faster than a for-loop. So that's good.

I do worry about the `isinstance()` check in the inner loop. Maybe it's possible to deal with this special case of one line outside the inner loop?

Reply:
previously, I passed every line to json.loads, which would through error on empty line, but now I only pass non-empty lines to json parser.

Reply:
The isinstance is only done once, to determine if the entire fs_data object is a string or a dict, then if it is a dict, put in a single item list (only a single line returned from borg), or split the string and convert each line to dict.

Reply:
True. Got it.